cli/src/main/scala/scalaxb/compiler/SbtApp.scala

```diff
@@ -16,7 +16,9 @@ class SbtApp extends xsbti.AppMain {
         logger.error(e.getMessage)
         Exit(1)
       case e: Exception =>
-        logger.error(e.getStackTrace.mkString("", Module.NL, Module.NL))
+        //logger.error(e.getStackTrace.mkString("", Module.NL, Module.NL))
+        val msg = e.getMessage
+        logger.error(if (msg == null) "" else (msg + "\n") + e.getStackTrace.mkString("", Module.NL, Module.NL))
```

Comment:
1. Plz remove the commented out line.
2. Null checking should be done using `(Option(e.getMessage) match { ... })`
3. When the msg is `null`, it will no longer log the stack trace because if-else is not in paren. See also 1.
4. `Module.NL` should be used instead of "\n"